digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/AssetService.scala

```diff
@@ -13,6 +13,24 @@ trait AssetOperations {
       assetDao.getMunicipalityById(id)
     }
   }
+
+  def getVerifiableAssets: Seq[String] = {
+    withDynTransaction {
+      assetDao.getVerifiableAssetTypes
```

Comment:
we have two methods with this name one in OracleAssetDao and another in VerificationDao